KoPrivateGPT/benchmark/retrival/base_retrieval_evaluation_factory.py

```diff
@@ -0,0 +1,255 @@
+from beir.retrieval import evaluation
```

Comment:
The package name 'retrival' is incorrect. Watch your spelling. 'retrieval is right spelling.

Reply:
thx.

actually, beir is not used in the code now